app/models/intake.rb

```diff
@@ -257,4 +257,13 @@ def external_id
 
     ["intake", id].join("-")
   end
+
+  def mixpanel_data
+    {
+      primary_filer_age_at_end_of_tax_year: primary_user&.age_end_of_tax_year.to_s,
+      spouse_age_at_end_of_tax_year: spouse&.age_end_of_tax_year.to_s,
+      primary_filer_disabled: had_disability,
+      spouse_disabled: spouse_had_disability,
```

Comment:
Nice, this will submit "unfilled" / "yes" / "no", right?

Reply:
yup!